packages/server/modules/core/services/streams/access.ts

```diff
@@ -217,3 +226,33 @@ export const addOrUpdateStreamCollaboratorFactory =
 
     return stream
   }
+
+export const setStreamCollaboratorFactory =
```

Comment:
annoying that add/remove are separate functions, this enables us to do both with 1 function